src/components/MainView.js

```diff
@@ -87,12 +97,42 @@ function MainView({ children }) {
           `}
         >
           <ScrollView>
-            <Layout
-              parentWidth={vw - (compactMode ? 0 : MENU_PANEL_WIDTH)}
-              paddingBottom={0}
+            <div
+              css={`
+                display: flex;
+                flex-direction: column;
+                min-height: 100%;
+              `}
             >
-              {children}
-            </Layout>
+              <div
+                css={`
+                  flex: 1 0 auto;
+                `}
+              >
+                <Layout
+                  parentWidth={vw - (compactMode ? 0 : MENU_PANEL_WIDTH)}
+                  paddingBottom={0}
+                >
+                  {children}
+                </Layout>
+              </div>
+              <footer
+                css={`
+                  flex-shrink: 0;
+                  margin-bottom: ${5 * GU}px;
+                  width: 100%;
+                  text-align: center;
+                  ${textStyle('body2')};
+                  color: ${theme.surfaceContentSecondary};
+                `}
+              >
+                Services provided by{' '}
+                <Link href="https://aragon.one">Aragon One.</Link> Read our{' '}
+                <Link href="https://anj.aragon.org/legal/terms-general.pdf">
+                  Terms of Service.
+                </Link>
```

Comment:
```suggestion
                <Link href="https://aragon.one">Aragon One</Link>. Read our{' '}
                <Link href="https://anj.aragon.org/legal/terms-general.pdf">
                  Terms of Service
                </Link>
                .
```

We leave the ending period out of the links :).